frontend/pages/[resourceType].tsx

```diff
@@ -104,8 +102,24 @@ export const getServerSideProps: GetServerSideProps<{
     // Measure Repository should not provide a bundle without an entry
     throw new Error('Measure Repository bundle has no entry.');
   }
-  const ids = bundle.entry.map(entry => entry.resource?.id);
+  const resources = bundle.entry as fhir4.BundleEntry<FhirArtifact>[];
+  const resourceInfoArray = resources.reduce((acc: ResourceInfo[], entry) => {
+    if (entry.resource && entry.resource.id) {
+      const identifier = entry.resource.identifier?.[0];
+      const resourceInfo: ResourceInfo = {
```

Comment:
If any of these values (like `name`) end up as nullish, it can't be serialized as server side props as JSON, you'll get this error:

<img width="1060" alt="image" src="https://user-images.githubusercontent.com/16297930/236019933-943e1950-df83-4df1-8708-d1416378e816.png">

Should probably default values to null or omit them from the object entirely if they're undefined

Reply:
@natjoe4 already identified this in a comment while I was reviewing lol